test/ios-project-service.ts

```diff
@@ -120,7 +137,7 @@ describe("Cocoapods support", () => {
 				"version": "0.1.0",
 				"nativescript": {
 					"id": "org.nativescript.myProject2",
-					"tns-android": {
+					"tns-ios": {
```

Comment:
maybe this change could be reverted after you fix the `tns-android` issue in `android-project-service.ts`